packages/table/src/V11Adapter/V11Adapter.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { ReactElement, useMemo, useRef, useState } from 'react';
+import { VirtualItem } from 'react-virtual';
+
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { Cell, HeaderCell } from '../Cell';
+import ExpandedContent from '../ExpandedContent/ExpandedContent';
+import HeaderRow from '../HeaderRow';
+import Row from '../Row';
+import TableBody from '../TableBody';
+import TableContainer from '../TableContainer';
+import TableHead from '../TableHead';
+import useLeafyGreenTable, {
+  LeafyGreenTableCell,
+  LeafyGreenTableRow,
+  LGColumnDef,
+  LGRowData,
+  LGTableDataType,
+} from '../useLeafyGreenTable';
+import Table, { flexRender, getCoreRowModel, getSortedRowModel } from '..';
+
+import processColumns from './processColumns';
+import processData from './processData';
+import { V11AdapterProps } from './V11Adapter.types';
+
+// assumes table is first element in children
```

Comment:
What happens if this is not the case?

Reply:
Since the V11Adapter is used pretty intentionally by the developer, I assume this should be an okay assumption to make. It's just a part of how the V11Adapter is expected to take in, like an API if you will.

Reply:
we should probably document this better then

Reply:
^ and check for errors